cmd/src/server.rs

```diff
@@ -579,22 +618,8 @@ fn pre_start(config: &TiKvConfig) {
     initial_logger(&config);
     tikv_util::set_panic_hook(false, &config.storage.data_dir);
 
-    // Print version information.
-    tikv::log_tikv_info();
-    info!(
-        "using config";
-        "config" => serde_json::to_string(&config).unwrap(),
-    );
-
-    config.write_into_metrics();
-
     check_system_config(&config);
```

Comment:
Could you move it into `init_config` as it requires configuration to perform check?

Reply:
We should leave `check_system_config` on `pre_start`, because `check_system_config` is mostly for checking system environment, and the config is checked on `validate_and_persist_config`.